src/libs/Navigation/Navigation.js

```diff
@@ -185,6 +189,16 @@ function isActiveRoute(routePath) {
     return path === routePath;
 }
 
+/**
+ * Returns the current active route
+ * @returns {String}
+ */
+function getActiveRoute() {
+    return navigationRef.current && navigationRef.current.getCurrentRoute().name
+        ? getPathFromState(navigationRef.current.getState(), linkingConfig.config)
+        : '';
+}
```

Comment:
Can `getActiveRoute()` be used in `isActiveRoute()` looks like there's some similar logic there?

Reply:
They are very similar, but I wasn't able to figure out a way to use it.

Reply:
Something like this... ?

```js
    // We remove First forward slash from the URL before matching
    return getActiveRoute().substring(1) === routePath;
```

Reply:
Ah oops, sorry I misread this comment, yes we can use `getActiveRoute` in `isActiveRoute`, I will update 👍